core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementEvent.java

```diff
@@ -262,6 +263,10 @@ void append() throws CommitFailedException {
                         row.putTimestamp(colIndex, buffer.readLong(offset));
                         offset += Long.BYTES;
                         break;
+                    case LineTcpParser.ENTITY_TYPE_UUID:
```

Comment:
We have a fairly similar "optimized" implementation for the WAL tables. `LineTcpMeasurementScheduler.appendToWal` should also be updated

Reply:
good one, thank you!

Reply:
implemented and pushed